src/components/organisms/Containers/TaskContainer/__tests__/TaskContainer.test.tsx

```diff
@@ -1,33 +1,36 @@
 import { fireEvent, screen, waitFor } from "@testing-library/react";
 import "@testing-library/jest-dom";
+import { MemoryRouter } from "react-router-dom";
 
 import TaskContainer from "../TaskContainer";
 import { renderWithState } from "utils/testHelpers";
 import { todoApi } from "services/todoAPI";
-import { Priority, TaskContent } from "types/type";
+import { Priority, Task, TaskContent, TaskFilters } from "types/type";
+
+const testTaskArray: Task[] = [
+  {
+    taskId: "1",
+    content: "lorem ipsum",
+    status: "active",
+    priority: Priority.p1,
```

Comment:
the Priority is an enum? If it is then it should be uppercase (such convention)

Reply:
Priority is an enum. Should I change every enum to uppercase?

Reply:
`enum Direction {
  Up = "UP",
  Down = "DOWN",
  Left = "LEFT",
  Right = "RIGHT",
}`
albo nawet lepiej
`enum Direction {
  UP = "UP",
  DOWN = "DOWN",
}`